js/main.js

```diff
@@ -1 +1,76 @@
 /* Javascript goes here! */
+console.log('is your script running? better go catch it!');
+
+$(document).ready(function(){
+  console.log('ready');
+
+//I DONT KNOW WHAT TO DO TO MAKE MY WEBSITE DELETE OLD STUFF ON ENTRY!!!
+  $('form').submit(function(event) {
+    event.preventDefault();
+    // $('.temperature').remove();
+    // $('.humidity').remove();
+    // $('.description').remove();
+
+    requestWeatherData();
+  });
+});
+
+function requestWeatherData(){
+  console.log('requesting data...');
+  $.ajax({
+    method: 'GET',
+    url: 'http://api.openweathermap.org/data/2.5/weather',
+    data: $('form').serialize() + '&units=imperial&appid=a77765c844f61df44fe43412c991b9cf',
```

Comment:
Since you are serializing the form, you can also add two hidden inputs for appid and units to the form to be included instead of writing them out by hand here.